assets/sass/_navigation.scss

```diff
@@ -84,6 +84,7 @@ Style guide: Navigation.Primary Navigation
     background-color: $background-colour;
     text-decoration: none;
     color: $non-black;
+    font-size: rem(14);
```

Comment:
I like the larger spacing that we have between nav list elements, but I’m not fully sold on the smaller font size? What was the rationale? (: